cluster/manifests/prometheus/service.yaml

```diff
@@ -4,12 +4,13 @@ metadata:
   name: prometheus
   namespace: kube-system
   labels:
-    application: prometheus
+    application: kubernetes
+    component: prometheus
 spec:
   type: ClusterIP
   ports:
     - port: 80
       targetPort: 9090
       protocol: TCP
   selector:
-    application: prometheus
+    component: prometheus
```

Comment:
Why not match both application+component?

Reply:
I expect a short downtime during rollout because it takes a while to roll the prometheus pods

I would add the `application` label in a follow up. (so this should be [3/4] really).

Reply:
Ah, I guess service should match pods before and after labels update